src/client/Makefile

```diff
@@ -8,7 +8,7 @@ RM = rm
 
 OUTPUT = nymphcast_client
 INCLUDE = -I .
-LIB := -lnymphrpc -lPocoNet -lPocoUtil -lPocoFoundation -lPocoJSON -lstdc++fs
+LIB := -lpthread -lnymphrpc -lPocoNet -lPocoUtil -lPocoFoundation -lPocoJSON -lstdc++fs
```

Comment:
-pthread is already set later on. This line would break building on Windows and other OSes.

Reply:
Right, ok